src/sinks/influxdb/logs.rs

```diff
@@ -188,15 +185,19 @@ impl HttpSink for InfluxDBLogsSink {
             }
         });
 
-        influx_line_protocol(
+        let mut output = String::new();
+        if let Err(error) = influx_line_protocol(
             self.protocol_version,
             measurement,
             "logs",
             Some(tags),
             Some(fields),
             timestamp,
             &mut output,
-        );
+        ) {
+            warn!(message = "Failed to encode event; dropping event.", %error, rate_limit_secs = 30);
```

Comment:
Should this be an event?

Reply:
This should be, but there is no internal metrics module for this sink yet so I'd prefer doing it separately.